### ISSUE TITLE
Having git mirror with branches/tags

### ISSUE THREAD

**Issue by insteps** [nmap/nmap#248]
Hi,

There are some more options to 'git svn'
eg
git svn clone https://svn.nmap.org/nmap -T trunk -b branches -t tags
or
git svn clone https://svn.nmap.org -T trunk -b branches -t tags

These options saves tags and branches info too.
Pls have a look at it and maybe still possible to redo the git mirror.
(Note, I have not tried on nmap repo)

**Comment by dmiller-nmap** [comment]
Related #412

**Comment by fyodor** [comment]
Good idea, but closing because we're hoping to move to a git-first setup and so we'd rather focus on that than improve the current svn to git mirroring system.
